tket/src/OpType/OpTypeJson.cpp

```diff
@@ -36,14 +36,19 @@ const std::map<std::string, OpType>& name_to_optype() {
 }
 
 void to_json(nlohmann::json& j, const OpType& type) {
-  j = optypeinfo().at(type).name;
+  j = optypeinfo().at(type).name;  // TODO wasm wire
+  const std::string name = j.get<std::string>();
+  const auto result = name_to_optype().find(name);
+  if (result == name_to_optype().end()) {
```

Comment:
I thought this op name "???" is generated here, but the same check that fails later in the "from_json" is not showing something.